invenio_app_ils/documents/covers_builder.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Document configuration callbacks."""
+
+
+def build_default_cover_urls(metadata):
+    """Build default ulrs for literature, documents & series."""
+    default_url = "/images/placeholder-portrait.png"
+    return {
+        "small": "{0}".format(default_url),
+        "medium": "{0}".format(default_url),
+        "large": "{0}".format(default_url),
+    }
+
+
+def build_ils_demo_cover_urls(metadata):
```

Comment:
👍 
Only one small remark: since you are re-using this for both `Documents` and `Series`, you might want to use the `ISBN` for `Documents` but the `ISSN` for `Series`. WDYT?
This has to be applied then in CDS Books with Syndetics.

Reply:
ISBN is for some multipart monographs, ISSN mostly for journals